Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM sonarsource/sonar-scanner-cli:5.0
+FROM sonarsource/sonar-scanner-cli:5.0.1
```

Comment:
Should be a separate change maybe?

Reply:
Do you mean having it in a separate commit or we will do the change later?